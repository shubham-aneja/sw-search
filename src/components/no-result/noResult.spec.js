import React from 'react';
import { mount } from 'enzyme';
import NoResult from './index.js'

describe('NoResult', () => {
    it('rendering', () => {
        const app = mount(<NoResult />)
        expect(app.html()).toMatchSnapshot();

    });

});

