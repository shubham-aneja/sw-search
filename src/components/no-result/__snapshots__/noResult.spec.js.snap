// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResult rendering 1`] = `"<div class=\\"no-result\\">No result found</div>"`;
